euth_wagtail/assets/scss/components/_dashboard-form-buttons.scss

```diff
@@ -0,0 +1,52 @@
+#dashboard-form-buttons {
```

Comment:
The main reasons against ids in CSS are:

- ids need to be unique on a page, so the styles attached to them are not very reusable
- ids have a infinitly higher specificity than anything else (except for `!important`). You should strive to keep a somewhat constant specificity across selectors in order to avoid surprising behavior.